runtime/jcl/common/java_lang_StackWalker.cpp

```diff
@@ -130,7 +130,7 @@ utfToStringObject(JNIEnv *env, J9UTF8 *utf, UDATA flags) {
 	if (NULL != utf) {
 		J9VMThread *vmThread = (J9VMThread *) env;
 		J9JavaVM *vm = vmThread->javaVM;
-		result = vm->memoryManagerFunctions->j9gc_createJavaLangString(vmThread, J9UTF8_DATA(utf), (U_32) J9UTF8_LENGTH(utf), flags);
+		result = vm->memoryManagerFunctions->j9gc_createJavaLangStringWithUTFCache(vmThread, utf, flags);
```

Comment:
Should `flags` be removed here?

Reply:
Yes.

Reply:
Actually, the call should not be changed to the new one as the flags are not the same for all callers.

Once https://github.com/eclipse-openj9/openj9/pull/13320 is merged, I will be removing this function entirely.